src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -127,14 +134,15 @@ export const isV1Artifact = _.curry((workspace, { googleProject, cloudContext })
 
 const WorkspaceContainer = ({
```

Comment:
Lots of changes here and in Dashboard module.... Dashboard module has a unit test file, but only testing WorkspaceNotifications currently.  Lets get some unit tests to at least cover the changed bits in these two files.

Reply:
The dashboard change are mostly just removing code that is no longer necessary because the functionality went into `useWorkspace`, which is fully covered by unit tests.

The dashboard has some existing coverage through the `workspace-dashboard` integration test; it covers the display of cloud environment location and other storage details, as well as which tabs are visible (WorkspaceContainer).

Reply:
Similarly, `WorkspaceContainer`'s changes are mostly removal of code that went into `useWorkspace`. The thing that isn't tested is the display of the banner during the time that the workspace is in the uninitialized period. I can think about how we would test that, but I do not want to hold this PR for it as IA needs to incorporate these changes into their Azure price display work. Honestly, I'm also not sure how to unit test WorkspaceContainer given all the arguments being passed around and the rendering code. Thankfully the file has gotten smaller at least!

Reply:
I am glad to hear about the existing coverage.  Yes, let's not hold up IA team, but maybe open up a follow-up ticket and we can pair on the unit testing outlook now that there will have been some helpful complexity refactoring.

Reply:
https://broadworkbench.atlassian.net/browse/WOR-773